<!--launch file for smoke package.-->
<launch>

  <!-- ROS and network parameter files -->
  <arg name="ros_param_file"  default="$(find smoke)/config/darknet_svm.yml"/>

  <!-- Load parameters -->
  <rosparam command="load" ns="smoke" file="$(arg ros_param_file)"/>
  
  <!--darknet_detetor-->
  <node
    pkg="smoke"
    type="darknet_detector"
    name="darknet_detector_node"
    output="screen"
    required="true"
  />

  <node
    pkg="smoke"
    type="kobuki_actSrv.py"
    name="kobuki_action"
    output="screen"
    required="true"
  />

  <node
    pkg="smoke"
    type="srvtest"
    name="service_test"
    output="screen"
    required="false"
  />

  <!--<node
    pkg="smoke"
    type="smoke_svm_client"
    name="smoke_svm_client_node"
    required="true"
  /-->

</launch>